ApollosProject/apollos-prototype
# Optimistic Response for Location Selector

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds an optimistic response to the Location Selector; this allows us to complete the screen as soon as we fire the mutation rather than waiting.

### What design trade-offs/decisions were made?

### How do I test this PR?

1. Go to the Location Selector
2. Select a new location, the screen should close immediately
3. Reopen the Location Selector, the location should reflect your change 

### What automated tests did you write?

## TODO:

- [ ] Closes #1231 

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
